Tests/varLib/varLib_test.py

```diff
@@ -204,6 +204,23 @@ def test_varlib_build_feature_variations(self):
             save_before_dump=True,
         )
 
+    def test_varlib_gvar_explicit_delta(self):
+        """The variable font contains a composite glyph n.001 which does not
```

Comment:
isn't the glyph in question named "odieresis" in the test ufo?

Reply:
Ah, that was from an older test ufo. Will fix.